web/styles/compose.css

```diff
@@ -677,6 +668,11 @@ input.recipient_box {
         padding-top: 0;
     }
 
+    .compose_gif_icon {
+        font-size: 22px;
+        display: flex;
```

Comment:
Since it doesn't look like `.compose_gif_icon` has any children, is there a need to set it to display as flex? Would displaying it as a block work just as well?